CI_JobHistory.py

```diff
@@ -241,9 +244,11 @@ def main():
     parser = argparse.ArgumentParser(description='Get the job history')
     parser.add_argument('--zone', help='specify the lease/zone', type= lambda arg:arg.split(','))
     parser.add_argument('--job_type', default='p', choices=['p','z','pa'], help= 'Specify the CI job type (Power(p) or s390x(z) or Power Auxillary(pa)), default is p')
+    parser.add_argument('--job_platform',default=' ')
```

Comment:
Can you add help statement to describe about the flag.

Reply:
Can you add what are the possible values of this flag in choices

Reply:
u can search any job_platform based on job config.keys

Reply:
okay. Can you add those possible values in help statement.

Reply:
PS C:\Users\AlokGoswami\project\ci-monitoring-automation> python3 .\CI_JobHistory.py --help                      
usage: CI_JobHistory.py [-h] [--zone ZONE] [--job_type {p,z,pa}] [--job_platform JOB_PLATFORM]

Get the job history

options:
  -h, --help            show this help message and exit
  --zone ZONE           specify the lease/zone
  --job_type {p,z,pa}   Specify the CI job type (Power(p) or s390x(z) or Power Auxillary(pa)), default is p
  --job_platform JOB_PLATFORM
                        enter job_platform as heavy libvirt powervs upgrade